### PR TITLE
fix(json_history): Append to log, instead of overwriting

### PR DESCRIPTION
**Pull request by haslersn**

### CODE REVIEW

File: src/server.rs [hunk 1]
```diff
@@ -37,6 +37,7 @@ pub async fn run(
         .manage(callback)
         .launch()
         .await
+        .map(drop)
```
Comment (neumantm) [inline src/server.rs]: What does this do?

**Review by neumantm — COMMENTED** [review]
Is now deployed and seems to work.

File: src/server.rs (hunk 1 above)
Comment (haslersn) [reply src/server.rs]: Rocket changed the return value from `()` to something else. `drop` is a function that takes any argument and returns `()`.

Comment (haslersn) [reply src/server.rs]: Now I changed the syntax to something more readable.

**Review by neumantm — APPROVED** [review]

**Pull request merged**